libevmasm/Instruction.cpp

```diff
@@ -223,6 +224,8 @@ static std::map<Instruction, InstructionInfo> const c_instructionInfo =
 	{ Instruction::COINBASE,	{ "COINBASE",		0, 0, 1, false, Tier::Base } },
 	{ Instruction::TIMESTAMP,	{ "TIMESTAMP",		0, 0, 1, false, Tier::Base } },
 	{ Instruction::NUMBER,		{ "NUMBER",			0, 0, 1, false, Tier::Base } },
+//	{ Instruction::PREVRANDAO,	{ "PREVRANDAO",		0, 0, 1, false, Tier::Base } },
+// TODO Replace with PREVRANDAO in the next breaking release
 	{ Instruction::DIFFICULTY,	{ "DIFFICULTY",		0, 0, 1, false, Tier::Base } },
```

Comment:
IMHO, this doesn't make sense.
It's a question of the EVM version, not with source compatibility - with the next breaking release compiling to <paris *still* should result in ``DIFFICULTY`` and already now, it's weird to emit a non-existent opcode name for >=paris.

So should really do this right away, as already described in https://github.com/ethereum/solidity/pull/13531#pullrequestreview-1109684090.

The EIP just changes the fact that there is a constant mapping from the binary opcode to instruction infos - we have to account for that in the code one way or another (and it's not like dragging an EVM version dependency through here is that much of a hassle).